src/groups/bmq/bmqp/bmqp_protocol.h

```diff
@@ -4134,12 +4138,13 @@ inline SchemaWireId::SchemaWireId()
 // PUBLIC MODIFIERS
 inline void SchemaWireId::set(unsigned value)
 {
-    d_value = value;
+    // Include precondition check or adjust function signature?
```

Comment:
Include precondition check or adjust function signature? Just remembered I left this comment in the commit; will likely remove this later